.github/workflows/codeql-analysis.yml

```diff
@@ -39,8 +39,8 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v4
 
-    - name: dotnet pack OpenTelemetry.proj
-      run: dotnet pack OpenTelemetry.proj --configuration Release
+    - name: dotnet pack
+      run: dotnet pack ./build/OpenTelemetry.proj --configuration Release
```

Comment:
Would this file move affect existing docs? e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md#pull-requests

Reply:
I don't think so. I searched the repo for all occurrences of `OpenTelemetry.proj` to make sure I wasn't breaking anything. What I will do (follow-up) is mention in that guide new files\projects should be added to the solution.